maraboupy/MarabouNetwork.py

```diff
@@ -287,3 +289,26 @@ def findError(self, inputValues, options=None, filename="evaluateWithMarabou.log
         outNotMar = self.evaluate(inputValues, useMarabou=False, options=options, filename=filename)
         err = np.abs(outMar - outNotMar)
         return err
+
+    def evaluateWithoutMarabou(self, inputValues):
```

Comment:
Defining this function in MarabouNetwork.py is not necessary, as it can be defined in the child class, MarabouNetworkNNet.py. See how evaluateWithoutMarabou() is already defined in the tensorflow and onnx parser files.

Reply:
Done.